backend/src/apps/api/serializers/ideas.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework import serializers
+from apps.ideas.models import Idea, IdeaTag
+
+class IdeaTagSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = IdeaTag
+        fields = ('id', 'name')
+
+class IdeaSerializer(serializers.ModelSerializer):
+    author_name = serializers.CharField(source='author.get_full_name', read_only=True)
+    status_display = serializers.CharField(source='get_status_display', read_only=True)
+    tags_data = IdeaTagSerializer(source='tag', many=True, read_only=True)
```

Comment:
@AlexZagric что думаешь насчет этого?

Reply:
Что не так? Назвать это лучше по другому конечно

```suggestion
    tags = IdeaTagSerializer(source='tag', many=True, read_only=True)
```